lib/plugins/transform/typeAdapter.js

```diff
@@ -120,6 +127,36 @@ module.exports = superclass => class extends superclass {
       .then(() => newValue);
   }
 
+  // object
+  objectValuesMergeAdapter(value) {
+    const newValue = {};
+    return Promise.all(Object.keys(value).map(key => super.transform(value[key])
+      .then(tmpValue => ({
```

Comment:
Is this `then` necessary? It wraps `tmpValue` in an object but the next then unwraps it and doesn't use the wrapped object.